build.gradle

```
@@ -144,14 +146,15 @@ publishToMavenLocal {
 }
 
 test {
+	dependsOn(gradle.includedBuild("TestPlugin").task(":jar"))
 	useJUnitPlatform()
 }
 
 eclipse.classpath.downloadJavadoc = true
 eclipse.classpath.downloadSources = true
 
 def isFork() {
-	try {
+	try {cdc
```

Hmm, this doesn't seem right.
I'm frankly surprised this even compiles (or maybe it won't once merged)

```suggestion
	try {
```